controllers/util/storageconsumer.go

```diff
@@ -107,6 +112,10 @@ func (wrapper storageConsumerResourceMapWrapper) GetNfsClientProfileName() strin
 	return wrapper.data[nfsClientProfileKey]
 }
 
+func (wrapper storageConsumerResourceMapWrapper) GetNoobaaAccountApplicability() string {
+	return wrapper.data[noobaaAccountKey]
+}
+
```

Comment:
I am not sure what do we need this for? And I am also failing to find where `GetNoobaaAccountName`/`SetNoobaaAccountName` are located?